www/static/js/dw/chart/publish.js

```diff
@@ -1,112 +0,0 @@
-
-define(function() {
-
-    var chart = dw.backend.currentChart;
-
-    function init(chartUrl, publish, republish) {
-        chart.sync('#embed-width', 'metadata.publish.embed-width');
-        chart.sync('#embed-height', 'metadata.publish.embed-height');
-
-        $('#embed-width, #embed-height').change(function() {
-            var w = $('#embed-width').val(), h = $('#embed-height').val(),
-                embedCode = $('#iframe-code').html();
-            embedCode = embedCode.replace(/width="\d+"/, 'width="' + w + '"');
-            embedCode = embedCode.replace(/height="\d+"/, 'height="' + h + '"');
-            $('#iframe-code').html(embedCode);
-
-            resizeIFrame();
-        });
-
-        $('.chart-actions .action-duplicate a').click(triggerDuplicate);
```

Comment:
if you delete this, the chart duplication UI will stop working

Reply:
thanks, fixed